scripts/scripts.js

```diff
@@ -698,6 +698,8 @@ export function getEnvType(hostname = window.location.hostname) {
     'www.sunstar-foundation.org': 'live',
     'main--sunstar-foundation--hlxsites.hlx.page': 'preview',
     'main--sunstar-foundation--hlxsites.hlx.live': 'live',
+    // Just for verification, it will be removed before merge to main branch
```

Comment:
You shouldn't need this. There's a fallback to `dev` if nothing matches.

Reply:
@sdmcraft I didn't get you. That line is only for verification purposes, I will remove it before merging. And, even the fallback returns `dev`, it cannot go into GTM logic because only `live` allows.

```
if (getEnvType() === 'live') {
    await loadGTM();
  }
}
```